spec/spec_helper.rb

```diff
@@ -30,8 +30,16 @@ def load_response(filename, hash: false)
   response
 end
 
-def load_request(filename)
-  File.read(File.join(MOCK_DIR, 'requests', filename))
+def load_request(filename, hash: false)
```

Comment:
Shouldn't this be `hash = false`?

Reply:
I'm not sure, in this file I mostly just copied method that @Misenko wrote above. But it works, tho.

Reply:
I'm too old for this shit! :-D

Reply:
This allows you to use hash (the data structure not the variable name 😄) as method argument and its keys will be available in the method as variables without explicitly setting them. Method call is more readable that way because you have `some_method(arg1, arg2: true)` instead of `some_method(arg1, true)` where you have no idea what the boolean argument does.